assets/src/scss/layout/_footer.scss

```diff
@@ -1,25 +1,30 @@
 .site-footer {
-  --footer-- {
+  _-- {
     font-family: $roboto;
     background: $dark-blue;
+    color: $white;
   }
-  padding: 45px 0 $n25;
   position: relative;
   z-index: 2;
   text-align: center;
   transition: margin 1s;
+  padding-top: 24px;
   clear: both;
   line-height: 0;
 
-  @include medium-and-up {
-    padding: $n25 0 $n20;
-  }
+  a {
+    transition: color 100ms linear;
 
-  @include large-and-up {
-    padding: 55px 0 $n25;
+    --site-footer--link-- {
```

Comment:
I think I earlier gave these a custom variable prefix because `a` is not very descriptive as name. On the other hand, this could also be solved by using custom labels in a UI that needs to have more readable names. The convenience of auto generating the name maybe outweighs the limited benefit of having `link` in the name, and if we use `a` consistently it can be easy to get used to.

@GP-Dan-Tovbein What do you think?